phpcs.xml

```diff
@@ -14,7 +14,6 @@
     key "config.platform.php".-->
     <config name="testVersion" value="7.1"/>
 
-    <exclude-pattern>*/docroot/*</exclude-pattern>
```

Comment:
Won't this also then do everything in `contrib` as part of the lint?

Reply:
we have `PHPCS_TARGETS` in `.env`